xml/admin_saltcluster.xml

```diff
@@ -207,6 +207,18 @@ placement:
 <screen>&prompt.cephuser;ceph-salt config /ceph_cluster/minions remove ses-min2</screen>
    </step>
   </procedure>
+  <important>
+    <para>
+      In the event of a failure and the minion you are trying to remove is
+      in permanently powered-off, you will need to remove the node from the
```

Comment:
> is in permanently powered-off

...?

I am not sure what is meant here. Either:
"is in a permanently-powered-off state"
or
"is permanently powered-off."

Reply:
Looks like I changed my mind halfway. Option A, sir.